src/main/java/com/danit/controllers/ContractController.java

```diff
@@ -38,4 +47,12 @@ public void deleteContractById(@PathVariable(name = "id") long id) {
     contractService.deleteContractById(id);
   }
 
+  @PostMapping("/contracts")
+  private String createContract(@RequestBody List<Contract> contracts) {
+    logger.info("Adding new contract");
+    contractService.bulkSaveContracts(contracts);
+    logger.info("Contract saved");
+    return "Contract saved";
```

Comment:
its a bad idea to return custom string as a "success" indicator, just return header with 200 status code